src/plugins/plan/components/Plan.vue

```diff
@@ -232,6 +233,7 @@ export default {
                 this.planObject = domainObject;
                 this.swimlaneVisibility = this.configuration.swimlaneVisibility;
                 this.planData = getValidatedData(domainObject);
+                this.setStatus(this.openmct.status.get(domainObject.identifier));
```

Comment:
is there a simple way to add coverage for this line

Reply:
This line should be covered by the e2e test I added? Wonder why that's not being picked up... 🤔